.gitignore

```diff
@@ -0,0 +1,21 @@
+## IDEs, editors, ...
+.DS_Store
+.idea
+.project
+*.bak
+*.iml
+*.sublime*
+*.swp
+/.settings
+
```

Comment:
I don't know why we are having multiple newlines here, but at this stage I'm afraid to ask.

Reply:
Book of scaffold. Chapter 1, verse 10

Reply:
This is a Wojtek thing, he loved four lines between everything because of some blog post.